cartographer/requests/jsonapi_flask_request_mixin.py

```diff
@@ -100,7 +100,12 @@ def get_bool(self, key):
     def get_json_api_version(self, default_version=None):
         if default_version is None:
             default_version = get_default_version()
-        return JSONAPIVersion(self.args.get('json-api-version', default_version))
+
+        json_api_version = self.args.get('json-api-version', default_version)
+        try:
+            return JSONAPIVersion(json_api_version)
+        except TypeError:
+            raise ParameterInvalid('json-api-version', json_api_version)
```

Comment:
I have no clue how that was added, I think it's a separate/different diff that got tacked in? I'll revert it.
Good catch